e2e-tests/fixtures/Plan.ts

```diff
@@ -538,17 +538,17 @@ export class Plan {
   }
 
   async waitForSchedulingStatus(status: Status) {
-    await expect(this.page.locator(this.schedulingStatusSelector(status))).toBeAttached();
+    await expect(this.page.locator(this.schedulingStatusSelector(status))).toBeAttached({ timeout: 10000 });
     await expect(this.page.locator(this.schedulingStatusSelector(status))).toBeVisible();
   }
 
   async waitForSimulationStatus(status: Status) {
-    await expect(this.page.locator(this.simulationStatusSelector(status))).toBeAttached();
+    await expect(this.page.locator(this.simulationStatusSelector(status))).toBeAttached({ timeout: 10000 });
```

Comment:
Does adding a timeout just give a quicker route to failure if the test isn't working?

Reply:
No. It looked like the expect timed out before the status got to the UI. This extends the timeout to hopefully give the backend enough time to finish simulating before the test tries to move on.